src/pages/index.js

```diff
@@ -32,71 +87,145 @@ const cardPopOut = new PopupWithImage({
 
 cardPopOut.setEventListeners();
 
-function createCard(item) {
-  const card = new Card(item, "#localeCard", openModalImage);
-  return card.generateCard();
+const deleteCardModal = new PopupWithDelete({
+  popupSelector: ".modal_type_delete-image",
+  handleBtnClick: () => {
+    api.deleteCard(cardToDelete.data._id);
+    cardToDelete.deleteCard();
+    deleteCardModal.close();
+  },
+});
+
+/*.then(cardToDelete.deleteCard())*/
+
+function handleDeleteClick(card) {
+  console.log(card);
+  deleteCardModal.open();
+  deleteCardModal.setEventListeners();
+  cardToDelete = card;
+  return cardToDelete;
+  /*
+  return card;
+  cardToDelete = card;
+  console.log(cardToDelete);
+  */
 }
 
-const firstCards = new Section(
-  { items: initialCards, renderer: createCard },
-  ".elements"
-);
+function handleLike() {
+  if (this.data.isLiked == false) {
```

Comment:
`this` should **not** be used **outside** classes in functions in `index.js` . This **context** is easy to lose. You need to find other ways of passing data.

**Please, correct it everywhere**